### PR TITLE
docs: update README files

### PR DESCRIPTION
**Pull request by gui-santos**
# Purpose of PR

I was going to add some information about the contribution models in GitHub, but now it feels unnecessary
Anyway, I updated the README files with the most up to date instructions I could remember
Also tried to change the format to make it more clear in some sections

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,52 +4,65 @@
 [![All Contributors](https://img.shields.io/github/all-contributors/contentful/forma-36/master)](#contributors-)
 [![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lernajs.io/)
 
-This is the monorepo for the Forma 36 design system by [Contentful](https://www.contentful.com).
+[Forma 36](https://f36.contentful.com/) is an open-source design system by [Contentful](https://www.contentful.com) created with the intent to reduce the overhead of creating UI by providing tools and guidance for digital teams building and extending Contentful products
```
Comment (denkristoffer) [inline README.md:7]: Is this official copy? It's a bit convoluted.

File: README.md [hunk 2]
```diff
@@ -4,52 +4,65 @@
...
+
 - [`forma-36-tokens`](https://github.com/contentful/forma-36/tree/master/packages/forma-36-tokens)
-  - Forma 36 design tokens. See the README [here](./packages/forma-36-tokens/README.md)
+  This package includes all the visual tokens (e.g. colors, margins, font sizes, etc) that used in the react library
 
-### Adding packages
+### Creating new packages
 
-You can use [Plop](https://plopjs.com/) to scaffold new component package. Run `yarn generate` and follow the steps in the CLI to create a component package. Using your input here, Plop will generate the relevant files and add the relevant imports/exports to the main `src/index.ts` file required to make the component available when publishing the library.
+We use use [Plop](https://plopjs.com/) create scripts that help you to scaffold new packages. On the root of the repo, you can run `yarn generate`. Then follow the steps in the CLI. Plop will generate the relevant files and add the relevant imports and exports to the main `src/index.ts` file required to make the component available when publishing the library.
```
Comment (denkristoffer) [inline README.md:48]: "We use Plop create" -> "We use Plop *to* create".

I'd also say "On the root of the repo" should use "In" or "At the root".

File: README.md (hunk 1 above)
Comment (gui-santos) [reply README.md:7]: This bit is in our website, I just copied and put the links

**Review by denkristoffer — APPROVED** [review]
LGTM 🙏

**Pull request merged**